app/services/orders/create_service.rb

```diff
@@ -4,8 +4,9 @@ class EarnedPointsException < StandardError; end
   class CreateService
```

Comment:
Couple suggestions regarding this ServiceObject:
-  I would name it `CreateOrderService` - otherwise later you can have 10 services named exactly the same - consider putting in in the Orders namespace as well
- Are you really creating the `Order` here? because it looks, like you are creating it in the controller, and then updating some parameters inside the service 
- naming of the helper methods is a bit misleading -? `not_enough_earned_points` - I would rather name it `check_employees_points_amount`, also payment_for_order should rather be something like `pay_for_order`

Reply:
I considered changing name to ""Create Order Service""" but I thought that a namespace is better option to simplification my code in service objects when the app will grow because I will have a separate logic for each module with simple actions.

I changed name from CreateService to SaveService.

I changed naming of the helper methods.